### PR TITLE
Linter improvements

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: pkg/compiler/native_test.go [hunk 1]
```diff
@@ -33,26 +33,17 @@ import (
...
-}
-
-// testPrintHash is a helper for updating contract hashes.
-func testPrintHash(u util.Uint160) {
-	fmt.Print(`"`)
-	for _, b := range u.BytesBE() {
-		fmt.Printf("\\x%02x", b)
-	}
-	fmt.Println(`"`)
+	require.Equalf(t, []byte(neo.Hash), cs.NEO.Hash.BytesBE(), "%q", string(cs.NEO.Hash.BytesBE()))
```
Comment (fyrchik) [inline pkg/compiler/native_test.go:36]: `%q` is bad in a sense that it doesn't escape all characters. Strange unicode glyphs are undesireable in interop code IMO. Can we just replace `testPrintHash` with `hashToHexString` then?

**Review by fyrchik — APPROVED** [review]

File: pkg/compiler/native_test.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/compiler/native_test.go:36]: I don't think it's a big problem if we end up with proper string representation of hash. But we can also use `%x` here, it'll look like this:
```
--- FAIL: TestContractHashes (0.00s)
    native_test.go:36: 
                Error Trace:    native_test.go:36
                Error:          Not equal: 
                                expected: []byte{0xf5}
                                actual  : []byte{0xf5, 0x63, 0xea, 0x40, 0xbc, 0x28, 0x3d, 0x4d, 0xe, 0x5, 0xc4, 0x8e, 0xa3, 0x5, 0xb3, 0xf2, 0xa0, 0x73, 0x40, 0xef}
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,3 +1,4 @@
                                -([]uint8) (len=1) {
                                - 00000000  f5                                                |.|
                                +([]uint8) (len=20) {
                                + 00000000  f5 63 ea 40 bc 28 3d 4d  0e 05 c4 8e a3 05 b3 f2  |.c.@.(=M........|
                                + 00000010  a0 73 40 ef                                       |.s@.|
                                 }
                Test:           TestContractHashes
                Messages:       f563ea40bc283d4d0e05c48ea305b3f2a07340ef
```

Comment (fyrchik) [reply pkg/compiler/native_test.go:36]: The whole point of the function was to "just copy the output to interop code" (and it hash is string there).

Comment (fyrchik) [reply pkg/compiler/native_test.go:36]: Not a big deal if hashes are stabilized, though.

Comment (roman-khimov) [reply pkg/compiler/native_test.go:36]: Then I'll leave it as "%q" for now, it should be sufficient.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**